templates/zerver/integrations/giphy.md

```diff
@@ -1,14 +1,15 @@
-# GIPHY GIF integration
+# Zulip GIPHY GIF integration
 
 Send animated GIFs with your message using GIPHY integration.
 
-![](/static/images/integrations/giphy/GIPHY_zulip.png)
+!!! tip ""
 
-GIPHY is enabled by default in Zulip Cloud. If the you're on a
-self-hosted server where the [GIPHY integration][help-center-giphy]
-isn't already configured, see the [configuration
-instructions][configure-giphy] to enable it.
+    GIPHY is enabled by default in Zulip Cloud. If you're on a
+    self-hosted server where the [GIPHY integration][help-center-giphy]
+    isn't already configured, see the [configuration
+    instructions][configure-giphy] to enable it.
 
+![](/static/images/integrations/giphy/GIPHY_zulip.png)
```

Comment:
This screenshot is going to get out of date every time the compose box buttons change ... so I think we should probably drop it. But I'd check on CZO to confirm, documentation channel.